transitions/index.js

```diff
@@ -95,8 +90,7 @@ var canRun = function(options) {
         transition &&
         typeof transition.filter === 'function' &&
         transition.filter(doc) &&
-        !isRevSame(doc) &&
-        !isV2XmlDataRecord(doc)
```

Comment:
Yeah develop already has transitions disabled by default, which has the same effect but still allows us to enable them when needed.  We need transitions for XML data records too, depending on the use case.